### DISCUSSION TITLE
emailjs.send() function not doing anything

### DISCUSSION THREAD

**Discussion by benphamroodman** [emailjs-com/help#89] [Q&A]
As far as I understand, I'm feeding the send() function the correct parameters. I'm triggering it inside a function of a react component: 

```
  sendmail() {
    
    let fullName = (document.getElementById("name").value);
    let userEmail = document.getElementById("email").value;
    let userPhone = document.getElementById("phone").value;
    let userMessage = document.getElementById("message").value;

    var contactParams = {
      from_name: fullName,
      from_email: userEmail,
      from_phone: userPhone,
      message: userMessage
    }

    emailjs.send('service_lyth09b', 'template_qt3665h', contactParams)
    .then((res) => {
      alert("Email sent!", res.status, res.text);
    }, (err) => {
      alert("Email failed to send.", err);
    });

    alert("Made it Here");
  }
```

When I trigger sendmail(), only the "Made it here" alert appears, no res or error alerts triggered by send(). What could I be missing here? Any help is appreciated, thanks!

**Comment by benphamroodman** [comment]
Maybe it goes without saying but I'm not receiving any emails either.

**Comment by xr0master** [comment]
Since you're using a form, you're probably not overriding the default behavior. So the page is reloaded.

**Reply by benphamroodman** [reply]
Thank you! What container should I be using if not a form? Or how can I override the default behavior?

**Reply by xr0master** [reply]
you may check this article https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault
